data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/handler/ArtifactRequestHandler.java

```diff
@@ -121,25 +122,34 @@ public boolean canHandle(@NotNull MultipartRequest multipartRequest) {
 
         DataAddress dataAddress = artifactRequestMessagePayload.getDataDestination();
 
+
+        var props = new HashMap<String, String>();
+        if (artifactRequestMessage.getProperties() != null) {
+            artifactRequestMessage.getProperties().forEach((k, v) -> props.put(k, v.toString()));
+        }
```

Comment:
The properties are attributed with `@JsonIgnore`. We should not use them, otherwise this functionally will be broken as soon as we serialize the Information Model class correctly.
Source:
https://github.com/International-Data-Spaces-Association/Java-Representation-of-IDS-Information-Model/blob/main/de/fraunhofer/iais/eis/ArtifactRequestMessageImpl.java

Reply:
The Infomodel will get donated to EDC, so we're in control of that anyways. So we're good.